### DISCUSSION TITLE
Hide Cancle, Create & add another, create button

### DISCUSSION THREAD

**Discussion by nirav-arsenaltech** [laravel/nova-issues#6363] [Q&A]
- Is there any way to Hide or remove Cancle, Create & add another, create buttons for one resource ?

![Screenshot (351)](https://github.com/laravel/nova-issues/assets/147684940/dfd02371-b9af-41e7-a8de-c621cf431501)

**Comment by kendalltristan** [comment]
I know this is an old question at this point, but I recently had the same question. I found a solution and figured I might as well put it here for anyone in the future who might need it. And if there's a better way to do it, I'm happy to hear it. Anyway, here's what I came up with:

First, create a new CSS file and then register it in the NovaServiceProvider's boot method:

`Nova::style('custom', public_path('css/nova.css'));`

In your new CSS file, use :has() and the "dusk" attributes to select the specific button you need based on the dusk value from the form panel. For instance, if your model is called "Profile" you'd do this:

`body:has(div[dusk='create-profile-panel']) button[dusk='create-and-add-another-button'] {
    display: none;
}`
